src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -249,14 +249,15 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
   static synthesizeAudioDevice(toneHz: number): Device {
     const audioContext = DefaultDeviceController.createAudioContext();
     const outputNode = audioContext.createMediaStreamDestination();
+    const gainNode = audioContext.createGain();
+    gainNode.gain.value = 0.0;
+    gainNode.connect(outputNode);
```

Comment:
Should this `gainNode.connect(outputNode)` be executed after the following `if` or it does not matter ?

Reply:
It doesn't matter. It just needs to get hooked up to the outputNode. If using a tone generator then that will then get hitched to the gainNode in the if-block.